scipy/stats/_qmc.py

```diff
@@ -1266,6 +1266,159 @@ def _random_oa_lhs(self, n: IntNumber = 4) -> np.ndarray:
         return oa_lhs_sample[:, :self.d]  # type: ignore
 
 
+class Hammersley(QMCEngine):
+    """Hammersley sequence.
+
+    The Hammersley sequence is the Halton sequence in dimensions :math:`n-1`
+    with a linear sweep in the last dimension.
+
+    Parameters
+    ----------
+    d : int
+        Dimension of the parameter space.
+    scramble : bool, optional
+        If True, use Owen scrambling for the Halton sequence and subtract
+        random offsets from the last dimension. Otherwise no scrambling is
+        done. Default is True.
+    optimization : {None, "random-cd", "lloyd"}, optional
+        Whether to use an optimization scheme to improve the quality after
+        sampling. Note that this is a post-processing step that does not
+        guarantee that all properties of the sample will be conserved.
+        Default is None.
+
+        * ``random-cd``: random permutations of coordinates to lower the
+          centered discrepancy. The best sample based on the centered
+          discrepancy is constantly updated. Centered discrepancy-based
+          sampling shows better space-filling robustness toward 2D and 3D
+          subprojections compared to using other discrepancy measures.
+        * ``lloyd``: Perturb samples using a modified Lloyd-Max algorithm.
+          The process converges to equally spaced samples.
+
+        .. versionadded:: 1.10.0
+    seed : {None, int, `numpy.random.Generator`}, optional
+        If `seed` is an int or None, a new `numpy.random.Generator` is
+        created using ``np.random.default_rng(seed)``.
+        If `seed` is already a ``Generator`` instance, then the provided
+        instance is used.
+
+    Notes
+    -----
+    The underlying Halton sequence has severe striping artifacts for even
+    modestly large dimensions. These can be ameliorated by scrambling.
+    Scrambling also supports replication-based error estimates and extends
+    applicabiltiy to unbounded integrands.
```

Comment:
I am not sure I understand what you mean.

Reply:
![image](https://user-images.githubusercontent.com/6570539/190840729-d74181e0-2d16-41ce-af55-2f437aafe2b4.png)

Reply:
lol indeed, seems like I could maybe think about rephrasing if I am confused with myself.